Makefile

```diff
@@ -25,12 +25,13 @@ setversion:
 	@echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
 	@echo "Please set a tag like:  git tag 3.17"
 
+POS = $(wildcard po/*.po)
 translate:
 	grunt nggettext_extract
-	for language in de nl fr cs ; do \
-		(cd po; msgmerge $$language.po template.pot > tmp.po; mv tmp.po $$language.po) ; \
+	for language in $(POS); do \
+		(msgmerge $$language po/template.pot > tmp.po; mv tmp.po $$language) ; \
```

Comment:
We can use `msgmerge -U --backup=off $$language po/template.pot` to avoid the temporary file

Reply:
Done in 848585bec87dd5ea115a64c70fa38fc43154c3db